utils/build.ps1

```diff
@@ -931,18 +931,18 @@ function Build-CMakeProject {
   $C = [Compiler]::Unspecified
   $CXX = [Compiler]::Unspecified
   if ($UseMSVCCompilers.Contains("C")) {
-    $C = [Compiler]::UseMSVC
+    $C = [Compiler]::MSVC
   } elseif ($UsePinnedCompilers.Contains("C")) {
-    $C = [Compiler]::UsePinned
+    $C = [Compiler]::Pinned
   } elseif ($UseBuiltCompilers.Contains("C")) {
-    $C = [Compiler]::UseBuilt
+    $C = [Compiler]::Built
   }
```

Comment:
Does it make sense to have this switch or to switch this as you were suggesting to `-UseCCompiler [Compiler]::{MSVC,Pinned,Built}` instead to avoid the switch? I don't know which would be easier.